gmso/lib/potential_templates.py

```diff
@@ -1,158 +1,130 @@
+import glob
+import os
+import json
+
 from gmso.core.potential import Potential
+from gmso.utils.singleton import Singleton
+from gmso.exceptions import GMSOError
 
+POTENTIAL_JSONS = glob.glob(os.path.join(os.path.dirname(__file__), 'jsons', '*.json'))
```

Comment:
I know the `os` module is the normal go to, but would it make sense to use [the `pathlib` module here?](https://docs.python.org/3/library/pathlib.html)
Ive been using them for a while now and they are quite nice, and i think they provide the same utility we use here, just a more higher-level api.

Reply:
Agreed.